aspnetcore/fundamentals/startup.md

```diff
@@ -1,75 +1,106 @@
 ---
-title: Application Startup in ASP.NET Core
+title: Application startup in ASP.NET Core
 author: ardalis
-description: Discover how the Startup class in ASP.NET Core configures services and the application's request pipeline. 
+description: Discover how the Startup class in ASP.NET Core configures services and the app's request pipeline. 
 keywords: ASP.NET Core,Startup,Configure method,ConfigureServices method
 ms.author: tdykstra
 manager: wpickett
-ms.date: 02/29/2017
+ms.date: 12/05/2017
 ms.topic: article
 ms.technology: aspnet
 ms.prod: asp.net-core
 uid: fundamentals/startup
 ---
-# Application Startup in ASP.NET Core
+# Application startup in ASP.NET Core
 
-By [Steve Smith](https://ardalis.com/) and [Tom Dykstra](https://github.com/tdykstra/)
+By [Steve Smith](https://ardalis.com), [Tom Dykstra](https://github.com/tdykstra), and [Luke Latham](https://github.com/guardrex)
 
-The `Startup` class configures services and the application's request pipeline.
+The `Startup` class configures services and the app's request pipeline.
 
 ## The Startup class
 
-ASP.NET Core apps require a `Startup` class, which is named `Startup` by convention. You specify the startup class name in the `Main` program's [WebHostBuilderExtensions](https://docs.microsoft.com/aspnet/core/api/microsoft.aspnetcore.hosting.webhostbuilderextensions) [`UseStartup<TStartup>`](https://docs.microsoft.com/aspnet/core/api/microsoft.aspnetcore.hosting.webhostbuilderextensions#Microsoft_AspNetCore_Hosting_WebHostBuilderExtensions_UseStartup__1_Microsoft_AspNetCore_Hosting_IWebHostBuilder_) method. See [Hosting](xref:fundamentals/hosting) to learn more about `WebHostBuilder`, which runs before `Startup`.
+ASP.NET Core apps require a `Startup` class, which is named `Startup` by convention. Specify the startup class with the [WebHostBuilderExtensions](/dotnet/api/Microsoft.AspNetCore.Hosting.WebHostBuilderExtensions) [UseStartup<TStartup>](/dotnet/api/microsoft.aspnetcore.hosting.webhostbuilderextensions.usestartup#Microsoft_AspNetCore_Hosting_WebHostBuilderExtensions_UseStartup__1_Microsoft_AspNetCore_Hosting_IWebHostBuilder_) method.
```

Comment:
With all the 2.0 work on WebHostBuilder the Startup class is slowing becoming unnecessary. That said it's still a convenient organization pattern and we don't have any plans to remove it from the templates.

Downgrade `requires` to `uses`?

Reply:
Sure ... will do on the next pass.